src/plots/rectanglePlot.ts

```diff
@@ -231,6 +231,30 @@ export module Plots {
       return this;
     }
 
+    /**
+     * Gets the Entities at a particular Point.
```

Comment:
We use capitals in comments when we refer to a class, but there is no class called `Entities`